KsApi/models/ManagePledgeViewBackingEnvelope.swift

```diff
@@ -0,0 +1,96 @@
+import Foundation
+
+public struct ManagePledgeViewBackingEnvelope: Swift.Decodable {
+  public var project: Project
+  public var backing: Backing?
+
+  public struct Project: Swift.Decodable {
+    public var id: String
```

Comment:
Aren't we moving away from `Int` IDs?

Reply:
I think it depends what we end up using `id` for - I know we've pretty much been using `id` all over the place to mean either `relayId` or just "kickstarter" `id`, but I wonder if we want to be more explicit with this naming moving forward so there's less confusion around what this value actually is 🤔 we could consider naming these properties `graphId` or `relayId` but it would require a refactor of some of our other models. Maybe just a comment for now is fine?